packages/mobile/locales/es-419/accountScreen10.json

```diff
@@ -8,6 +8,7 @@
   "removeAccountDetails": "Al reinicializar se borrará tu cuenta de este dispositivo. Tu cuenta mantendrá los fondos que tenga, pero solo la podrás recuperar con tu Clave de Cuenta.",
   "legal": "Legal",
   "securityAndData": "Seguridad y datos",
+  "changePIN": "Cambiar PIN",
```

Comment:
We shouldn't add translations for strings, that's handled by Crowdin now

Reply:
oh ok! how does Crowdin handle translations? is there anything i have to add for that?

Reply:
No need to add anything, it happens automatically. I don't know the details too well, but whenever a new string is pushed to `main` Crowdin picks it up and asks in its UI (which is used by the product team afaik) for translations. Whenever we want to sync what's on Crowdin with the code we can tell Crowdin to create a PR with all the translation changes.

Reply:
We do need to delete the `es` and `pt` translations though because we might get conflicts when Crowding opens its PR otherwise

Reply:
oh nice!! thanks :)